elements/pfe-modal/src/pfe-modal.scss

```diff
@@ -0,0 +1,142 @@
+@import "../../pfe-sass/pfe-sass";
+$LOCAL: modal;
+
+$pfe-modal--breakpoint--medium: 640px;
+
+:host {
+	--pfe-modal--MaxWidth: 70vw;
+	--pfe-modal--MinWidth: 50vw;
+	--pfe-modal--MaxWidth--mobile: 94vw;
+	--pfe-modal--MaxHeight: 90vh;
+	--pfe-modal--Padding: calc(#{pfe-var(container-padding)} * 2) 0 calc(#{pfe-var(container-padding)} * 2) calc(#{pfe-var(container-padding)} * 2);
+	--pfe-modal--Padding--mobile: 0 0 0 calc(#{pfe-var(container-padding)} * 2);
+	--pfe-modal__overlay--BackgroundColor: #{pfe-color(overlay)};
+	--pfe-modal--CloseButtonSize: calc(#{pfe-var(ui--element--size)} - 4px);
```

Comment:
Can we rename this to: `--pfe-modal--CloseButtonSize` to `--pfe-modal__close--size`?  In BEM naming this would be `__{region}--{abstract name}`.  The only time BEM uses camel case is for CSS properties like `BackgroundColor` or `Padding`.

Reply:
Good one. I altered the variable name slightly `-pfe-modal--close--size` so that it would work with the `pfe-local` function. Let me know if I should change it back.

Reply:
the `__close` is the region which is what the underscore highlights.  You can do this in the pfe-local function like this: `pfe-local($cssvar: size, $region: close)`.

Reply:
Ahh, thanks. Done.